mcux/README

```diff
@@ -60,4 +60,9 @@ License Link:
 Patch List:
   1. Delete the .a binary archive files after an SDK update before committing the changes. These files are still present in the SDK github repo but have been
      deleted from the Zephyr HAL
-  2. Move used job descriptors in the CAAM driver (mcux-sdk/drivers/caam/fsl_caam.c) from the stack to noncacheable section. At time of writing, there should be four being used for entropy in zephyr.
+  2. Move used job descriptors in the CAAM driver (mcux-sdk/drivers/caam/fsl_caam.c) from the stack to noncacheable section. At time of writing, there should
+     be four being used for entropy in zephyr.
+  3. Updated defines to support FlexCAN-FD from zephyr (mcux/hal_nxp.cmake).
```

Comment:
No need for this item. We maintain this list to track local changes to the SDK files so that we don't lose them during SDK updates.

Reply:
Ah, I didn't get that. Do I revert the formatting of point 2?

Reply:
Just point 3 can be deleted.

Reply:
All right, done.